### ISSUE TITLE
A bug: batchsize>1 it goes wrong!

### ISSUE THREAD

**Issue by Zhaoyi-Yan** [Zhaoyi-Yan/Shift-Net_pytorch#24]
@tchaton 
I DO NOT when it goes like  it.
When set `batchSize=8` and run `acc_unet_shift`.
When batchsize>1, `x_latter.size()` is `(8, 128, 64, 64)`, while `x.size()` is `(1, 128, 64, 64)`

```
  File "/home/yan/github/Shift-Net_pytorch/models/modules/shift_unet.py", line 236, in forward
    return torch.cat([x_latter, x], 1)  # cat in the C channel
RuntimeError: invalid argument 0: Sizes of tensors must match except in dimension 1. Got 8 and 1 in dimension 0 at /opt/conda/conda-bld/pytorch_1535488076166/work/aten/src/THC/generic/THCTensorMath.cu:87
```

**Comment by tchaton** [comment]
@Zhaoyi-Yan 

I will have a look into that :)
And also the multi-gpu error.

Best,

**Comment by Zhaoyi-Yan** [comment]
@tchaton 

I think this is not true.
https://github.com/Zhaoyi-Yan/Shift-Net_pytorch/blob/9ef939407e78d7be6c477302a244b1f32c4fae4e/models/accelerated_shift_net/accelerated_InnerShiftTripleFunction.py#L52

When batchsize is 1, it goes well.
However, batchsize is 9, then we just concat `8*256*32*32`, `8*256*32*32` with `1*256*32*32`

**Comment by Zhaoyi-Yan** [comment]
Have fixed via d2c500803a9d7141a3933d0f4804d9635cd7bfc6
